src/pages/docs/run-anywhere/adding-provisioners-and-admins.mdx

```diff
@@ -44,22 +44,22 @@ To review the configuration for all existing provisioners on your certificate au
 step ca provisioner list
 ```
 
-Check the existing configuration for a given provisioner by running [`step beta ca provisioner get`](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/get):
+Check the existing configuration for a given provisioner by running [`step ca provisioner get`](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/get):
```

Comment:
There is no `get` command for `step ca provisioner`. And it's using a beta URL link.